### DISCUSSION TITLE
Update vscode

### DISCUSSION THREAD

**Discussion by rachadnet** [coder/code-server#2796] [Q&A]
How to update vscode to vscode 1.5. On Android. I am using termux and a version of ubuntu.

**Comment by code-asher** [comment]
To upgrade just install code-server over the old version: https://github.com/cdr/code-server/blob/v3.9.0/docs/install.md#upgrading

So for example if you installed with `yarn` you should be able to upgrade with `yarn global add code-server` (or `yarn global upgrade`).
